fontir/src/ir.rs

```diff
@@ -480,13 +501,24 @@ impl NameBuilder {
                 .unwrap()
                 .replace("Version ", "");
             // fontmake pulls the openTypeOS2VendorID but we don't have that so just use their default
-            let vendor = "NONE";
             let postscript_name = self.get(NameId::POSTSCRIPT_NAME).unwrap();
             self.add(
                 NameId::UNIQUE_ID,
-                format!("{version};{vendor};{postscript_name}"),
+                format!("{version};{vendor_id};{postscript_name}"),
             );
         }
+
+        // based on diffing with fontmake
```

Comment:
I'd avoid comments like these, it suggests we're treating fontmake as a black box which shouldn't be the case. We should either point directly to the source code that does the equivalent of what is below, or add a link to the spec, or if it is not self evident try to come up with some rationale. And if it still doesn't make any sense, raise an issue.

Reply:
Fair point, fixed in #323